core/src/main/scala/org/apache/spark/SparkException.scala

```diff
@@ -87,6 +87,16 @@ object SparkException {
       messageParameters = Map("message" -> msg),
       cause = cause)
   }
+
+  /**
+   * Tests an condition, throwing an `SparkException` with
+   * the error class `INTERNAL_ERROR` if false.
+   */
+  def checkInternalError(condition: Boolean, msg: String): Unit = {
```

Comment:
Is this tiny utility method worth it? not sure

Reply:
I want to be as close as possible to the existing `require` and don't introduce unnecessary boilerplate code.